tests/unit/explainer/test_visualization.py

```diff
@@ -178,14 +182,21 @@ def test_visualizer(
                 1: Prediction(bounding_box=[2, 5, 9, 7], score=0.5, label=0),
             }
             explanation = Explanation(saliency_maps, targets=-1, task=task, predictions=predictions)
+
             visualizer = Visualizer()
-            explanation_output_size = visualizer(
+            explanation = visualizer(
                 explanation=explanation,
                 original_input_image=original_input_image,
-                output_size=(20, 20),
                 scaling=scaling,
                 resize=resize,
                 colormap=colormap,
                 overlay=overlay,
                 overlay_weight=overlay_weight,
+                overlay_prediction=overlay_prediction,
             )
+
+            if task == Task.CLASSIFICATION and original_input_image.shape[0] == 100 and overlay:
+                if overlay_prediction:
+                    assert np.all(explanation.saliency_map[0][10, 6] == np.array([255, 0, 0], dtype=np.uint8))
+                else:
+                    assert np.any(explanation.saliency_map[0][10, 6] != np.array([255, 0, 0], dtype=np.uint8))
```

Comment:
Since we control input and saliency maps, we can compare the pixel with the actual one, right?